internal/cli/apis.go

```diff
@@ -38,6 +38,20 @@ var (
 		Help:       "Comma-separated list of scopes.",
 		IsRequired: true,
 	}
+	apiTokenLifetime = Flag{
+		Name:       "Token-Expiration",
+		LongForm:   "token-expiration",
+		ShortForm:  "t",
+		Help:       "The amount of time in seconds that the token will be valid after being issued",
+		IsRequired: true,
```

Comment:
```suggestion
		IsRequired: false,
```
Sorry, I did not include this on the card.